WalletWasabi.Backend/WalletWasabi.Backend.csproj

```diff
@@ -38,11 +38,28 @@
   <ItemGroup>
     <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.6" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="5.0.7" />
```

Comment:
5.0 packages in 3.1 is gonna cause some headaches. I fixed this in https://github.com/DanGould/WalletWasabi/pull/7